### PR TITLE
Type declarations for a new non-npm package sinch-rtc

### PR DESCRIPTION
**Pull request by dbagia**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/sinch-rtc/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        Ended = 3,
+    }
+
+    enum Direction {
+        Inbound = 0,
+        Outbound = 1,
+    }
+
+    class CallDetails implements CallListener {
+        private session;
```
Comment (peterblazejewicz) [inline types/sinch-rtc/index.d.ts:76]:
```suggestion
```
you've private here and there. What's the purpose? Those won't be usable (and are not visible visible in linked docs)

**Comment by sandersn** [comment]
How do these types relate to the existing npm package `sinch-rtc`. That looks like it's provided on npm for development, which sounds like it would be a good basis for types. Would it make sense to change the name of this package to sinch-rtc and make the types match?

File: types/sinch-rtc/index.d.ts (hunk 1 above)
Comment (dbagia) [reply types/sinch-rtc/index.d.ts:76]: Thanks for pointing out.  I will remove those and update this PR 👍

**Comment by dbagia** [comment]
@sandersn - To be honest, I wasn't aware there is an existing npm package `sinch-rtc` but we realise that it is not relevant anymore and we will remove it. These types are for a non-npm package that is downloadable through cdn.

File: types/sinch-rtc/index.d.ts (hunk 1 above)
Comment (dbagia) [reply types/sinch-rtc/index.d.ts]: Fixed

**Comment by sandersn** [comment]
Specifically, the folder name needs to be sinch-rtc-browser to match the header name.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
Folder name needs to be `sinch-rtc-browser `

**Comment by dbagia** [comment]
Thanks @sheetalkamat. I have a question about the folder renaming. We are in the process of deleting the existing `sinch-rtc` package from npm. Do you think that I still need to rename this package to `sinch-rtc-browser`? Is the convention relevant because this is just the browser package?

File: types/sinch-rtc/tslint.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
```
Comment (sheetalkamat) [inline types/sinch-rtc/tslint.json:3]: This needs to be removed and error needs to be fixed

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

**Comment by sheetalkamat** [comment]
The types cannot be for non npm package and have same name as package on npm. The folder name needs to match with the header in index,d,ts

File: types/sinch-rtc/tslint.json (hunk 2 above)
Comment (dbagia) [reply types/sinch-rtc/tslint.json]: Fixed

**Comment by dbagia** [comment]
I have now added `-browser` to the package name. It will be great someone can have a look again. We are also in the process of deleting the existing npm package `sinch-rtc` from npm but I believe it is not relevant after this renaming.

File: types/sinch-rtc-browser/index.d.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // Type definitions for non-npm package sinch-rtc-browser 2.31
 // Project: https://developers.sinch.com/docs/in-app-calling/js-cloud/
-// Definitions by: Johan Aschan <https://github.com/sinch-johana>
+// Definitions by: Sinch Johana <https://github.com/sinch-johana>
```
Comment (peterblazejewicz) [inline types/sinch-rtc-browser/index.d.ts:3]: this person has to give ACK to be listed here, DT sends notifications
to package owners:
https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
Please fix the owner name per feedback

**Review by sandersn — APPROVED** [review]

**Comment by sinch-johana** [comment]
@sandersn hi we found some problem with this PR. Can you revert it from master? Or what is the process to do that?

**Pull request merged**